pkg/reconciler/v1alpha1/taskrun/entrypoint/entrypoint.go

```diff
@@ -187,6 +187,13 @@ func getRemoteImage(image string, kubeclient kubernetes.Interface, build *buildv
 		return nil, fmt.Errorf("Failed to parse image %s: %v", image, err)
 	}
 
+	// First try to get the image anonymously
+	// FIXME(vdemeester): once google.Keychain fails smoother, this could be removed
```

Comment:
Link to a description of how it fails badly now, which will let future readers know whether this can be cleaned up.